app/src/main/res/values/strings.xml

```diff
@@ -177,26 +177,26 @@
-    <string name="restore_restoring_error_message">You can re-install these apps manually and "Automatic restore" will attempt to restore their data (when enabled).</string>
-    <string name="restore_magic_package">System package manager</string>
-    <string name="restore_finished_success">Restore complete</string>
-    <string name="restore_finished_error">An error occurred while restoring the backup.</string>
+    <string name="restore_restoring_error_message">You can re-install these apps manually and "Auto-restore" will attempt to restore their data (when on).</string>
+    <string name="restore_magic_package">System package-manager</string>
+    <string name="restore_finished_success">Restoration complete</string>
+    <string name="restore_finished_error">Could not restore the backup.</string>
     <string name="restore_finished_button">Finish</string>
 
     <string name="restore_storage_skip">Skip restoring files</string>
     <string name="restore_storage_choose_snapshot">Choose a storage backup to restore (experimental)</string>
-    <string name="restore_storage_in_progress_title">Files are being restored…</string>
-    <string name="restore_storage_in_progress_info">Your files are being restored in the background. You can start using your phone while this is running.\n\nSome apps (e.g. Signal or WhatsApp) might require files to be fully restored to import a backup. Try to avoid starting those apps before file restore is complete.</string>
+    <string name="restore_storage_in_progress_title">Restoring files…</string>
+    <string name="restore_storage_in_progress_info">You can use your device while this is running in the background.\n\nSome apps (e.g. Signal or WhatsApp) might require files to be fully restored to import a backup. Try to avoid starting those apps before file restoration completes.</string>
     <string name="restore_storage_got_it">Got it</string>
 
     <string name="storage_internal_warning_title">Warning</string>
-    <string name="storage_internal_warning_message">You have chosen internal storage for your backup. This will not be available when your phone is lost or broken.</string>
+    <string name="storage_internal_warning_message">Your choice of storing backups internally means they are not available if your device is lost or broken.</string>
     <string name="storage_internal_warning_choose_other">Choose other</string>
     <string name="storage_internal_warning_use_anyway">Use anyway</string>
 
     <!-- About -->
     <string name="about_title">About</string>
-    <string name="about_summary">A backup application using Android\'s internal backup API.</string>
+    <string name="about_summary">A backup app using Android\'s internal backup API.</string>
     <string name="about_version">Version: %s</string>
     <string name="about_license">License: <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache2</a></string>
```

Comment:
```suggestion
    <string name="about_license">License: <a href="https://www.apache.org/licenses/LICENSE-2.0">Apache 2.0</a></string>
```

Reply:
`License:` is pretty standard, no?

`Apache 2.0` would be correct.

Reply:
I would go "Libre software, licensed Apache 2.0", and then add "use, study, change and share freely", postponed mentally into 2026 :)